test_scripts/API/VehicleData/Refactor_Fuel_Information/002_GetVehicleData_with_one_param_with_boundary_value_in_fuelRange_strucrure.lua

```diff
@@ -0,0 +1,31 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
File should be renamed to `002_GetVehicleData_with_one_param_with_boundary_value_in_fuelRange_structure.lua`